packages/main/src/Input.js

```diff
@@ -1076,6 +1076,9 @@ class Input extends UI5Element {
 				"ariaExpanded": this._inputAccInfo && this._inputAccInfo.ariaExpanded,
 				"ariaDescription": this._inputAccInfo && this._inputAccInfo.ariaDescription,
 				"ariaLabel": (this._inputAccInfo && this._inputAccInfo.ariaLabel) || getEffectiveAriaLabelText(this),
+				"min": this._inputAccInfo.min,
```

Comment:
Those attributes are not accessibility related and should not be used as part of _inputAccInfo. They are regular attributes for a certain type of input.

Reply:
Done